.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+name: 💡 Feature Request
+description: Suggest an idea for EmbassyOS
+title: '[feat]: '
+labels: ["Bug", "Needs Triage"]
```

Comment:
These labels wouldn't be appropriate for a feature request.  "Enhancement" is the only one that comes to mind